### PR TITLE
Initial project skeleton

### PR DESCRIPTION
**Pull request by Robchaves**
Initial configuration of the project

### CODE REVIEW

**Review by luvalverde — COMMENTED** [review]
Like this will be the base of the project, would be useful to define directories to separate:
- header files
- source code
- tests

so, the following code will be located in an standard place

**Comment by Robchaves** [comment]
Changes applied please review @luvalverde  @wfallascr

File: localMatchApp/logIn.h [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+{
+public:
+  logIn();
+  ~logIn();
+  logIn ( const logIn & ) = default;
+  logIn &  operator= ( const logIn & ) = default;
+
+  int signUp(const std::string i_user, const std::string i_password);
+
+  bool isEqualPass(const std::string i_password, const std::string i_repeatedPassword);
```
Comment (luvalverde) [inline localMatchApp/logIn.h]: these param names are not equal as in localMatchApp/logIn.cpp
please change those names as they are in this file

**Review by luvalverde — CHANGES_REQUESTED** [review]
one more comment

File: localMatchApp/logIn.h (hunk 1 above)
Comment (Robchaves) [reply localMatchApp/logIn.h]: Done

File: localMatchApp/Sources/logIn.cpp [hunk 2]
```diff
@@ -0,0 +1,71 @@
+#include "logIn.h"
+#define OCUPATED_USER 3
```
Comment (luvalverde) [inline localMatchApp/Sources/logIn.cpp]: these error messages would be better to be contained in an enum

File: localMatchApp/Sources/logIn.cpp [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    //Disconnect to database.
+}
+
+int logIn::signUp(const string i_user, const string i_password, const string i_repeatedPassword)
+{
+  //Send information to database.
+  string userInDatabase="";
+  if(i_user.compare(userInDatabase)==0)
+    {
+      if(isEqualPass(i_password,i_repeatedPassword))
```
Comment (luvalverde) [inline localMatchApp/Sources/logIn.cpp:21]: separate params with an space

**Review by luvalverde — CHANGES_REQUESTED** [review]
some comments

File: localMatchApp/Sources/logIn.cpp (hunk 3 above)
Comment (Robchaves) [reply localMatchApp/Sources/logIn.cpp:21]: Done

File: localMatchApp/Sources/logIn.cpp (hunk 2 above)
Comment (Robchaves) [reply localMatchApp/Sources/logIn.cpp]: Done

File: localMatchApp/Headers/logIn.h [hunk 4]
```diff
@@ -0,0 +1,18 @@
+#include <string>
+
+
+class logIn
+{
+private:
+  bool isEqualPass(const std::string i_password, const std::string i_repeatedPassword);
+  enum connectionMessage { wrongPassword, goodEntry, doesntExists, ocupatedUser};
```
Comment (luvalverde) [inline localMatchApp/Headers/logIn.h:8]: According with the coding standard, use C++11 scoped enumerations.
Also, it is recommended to place only one expression on each line.

File: localMatchApp/Headers/logIn.h [hunk 5]
```diff
@@ -0,0 +1,18 @@
+#include <string>
+
```
Comment (luvalverde) [inline localMatchApp/Headers/logIn.h:2]: remove this blank line

File: localMatchApp/Sources/logIn.cpp [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+logIn::~logIn()
+{
+    //Disconnect to database.
+}
+
+int logIn::signUp(const string i_user, const string i_password, const string i_repeatedPassword)
+{
+  //Send information to database.
+  string userInDatabase="";
```
Comment (luvalverde) [inline localMatchApp/Sources/logIn.cpp:18]: it is more clear to initialize it like:
```c
string userInDatabase = "";
```

apply this same comment in all the places

File: localMatchApp/Sources/logIn.cpp [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+    }
+}
+
+int logIn::entry(const string i_user, const string i_password)
+{
+  //Search the user and compare password
+  //Query asking for password based on user
+  string userInDatabase="";
+  string passInDatabase="";
+  if(i_user.compare(userInDatabase) != 0)
```
Comment (luvalverde) [inline localMatchApp/Sources/logIn.cpp:56]: please add a blank line here

**Review by luvalverde — CHANGES_REQUESTED** [review]
Some minor comments @Robchaves

**Comment by Robchaves** [comment]
Compile problems that paused the production

**Pull request closed without merging**